examples/with-apollo-neo4j-graphql/apollo/resolvers.js

```diff
@@ -2,14 +2,7 @@ import { neo4jgraphql } from 'neo4j-graphql-js'
 
 export default {
   Query: {
-    getMovies: (parent, args, context, resolveInfo) => {
```

Comment:
Are they set up this way on purpose, though? To allow you to add more stuff inside these functions?